src/gluonts/model/tpp/forecast.py

```diff
@@ -107,21 +107,24 @@ def __init__(
         self.start_date = start_date
 
         assert isinstance(freq, str), "freq should be a string"
-        self.freq = freq
+        self.interarrival_freq = freq
```

Comment:
Why not directly use `freq`?

Reply:
right I thought it was a better name for this arg. rather than the data freq.... I can revert if you like?

Reply:
I guess either are fine. Let's just merge this and think about it later (maybe).